pkg/kubelet/kuberuntime/labels.go

```diff
@@ -38,6 +39,8 @@ const (
 	containerTerminationMessagePolicyLabel = "io.kubernetes.container.terminationMessagePolicy"
 	containerPreStopHandlerLabel           = "io.kubernetes.container.preStopHandler"
 	containerPortsLabel                    = "io.kubernetes.container.ports"
+
+	windowsHostProcessContainer = "microsoft.com/hostprocess-container"
```

Comment:
I see the discussion that this is temporary label. I think the comment in code will help to highlight this.

Reply:
added comment